src/AppBundle/Form/Type/DateTimePickerType.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Form\Type;
+
+use AppBundle\Utils\MomentFormatConverter;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Defines the custom form field type used to manipulate datetime values across
+ * Bootstrap Date\Time Picker javascript plugin.
+ * See http://symfony.com/doc/current/cookbook/create_custom_field_type.html
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class DateTimePickerType extends AbstractType
+{
+    /**
+     * @var MomentFormatConverter
+     */
+    private $formatConverter;
+
+    public function __construct()
+    {
+        $this->formatConverter = new MomentFormatConverter();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        $view->vars['attr'] = array_merge($view->vars['attr'], array(
```

Comment:
@yceruto, Passing those attributes can break the form, isn't it? I think you should use options 'format' and 'locale' for this.